hw/ip/rstmgr/data/rstmgr_testplan.hjson

```diff
@@ -0,0 +1,40 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  name: "rstmgr"
+  // TODO: remove the common testplans if not applicable
+  import_testplans: ["hw/dv/tools/dvsim/testplans/csr_testplan.hjson",
+                     "hw/dv/tools/dvsim/testplans/mem_testplan.hjson",
```

Comment:
I do not think there is any mem in rstmgr, maybe we can remove this?

Reply:
`alert_test_testplan` should be added here

Reply:
Done.